source/includes/steps-create-local-master-key.yaml

```diff
@@ -0,0 +1,68 @@
+title: Create the local master key
+ref: create-local-master-key
+level: 4
+content: |
+  Generate the local master key and save it to a file. You should only need to run this once:
+
+  .. tabs::
+
+     tabs:
+
+       - id: java-master-key-generator
+         name: "Java"
+         content: |
+
+           .. code-block:: java
+
+             import java.io.FileOutputStream;
+             import java.io.IOException;
+             import java.security.SecureRandom;
+
+             public class CreateMasterKeyFile {
+               public static void main(final String[] args) {
+
+                 final byte[] localMasterKey = new byte[96];
+                 new SecureRandom().nextBytes(localMasterKey);
+
+                 String path = "master-key.txt";
+                 try (FileOutputStream stream = new FileOutputStream(path)) {
+                   stream.write(localMasterKey);
+                 } catch (IOException e)  {
+                   e.printStackTrace();
+                 }
+               }
+             }
+
+---
+title: Load the local master key into the client settings 
```

Comment:
I'm not sure that this is really "loading into the client settings" - we don't have any MongoClient code or JSON Schema in this. I like your wording in the step content: **Construct the KMS Settings Object**

Reply:
Resolved in-person, moving settings to client setup section